src/www/ui/api/documentation/openapiv2.yaml

```diff
@@ -776,10 +776,45 @@ paths:
                 fileInput:
                   type: string
                   format: binary
+                folderId:
+                  type: integer
+                  description: Folder Id, where upload should be created
+                uploadDescription:
+                  type: string
+                  description: Visible description of the file
```

Comment:
Instead of repeating the properties, you can create a component in the schema and refer to it here (as used by `scanOptions` for example).

Reply:
It will prevent further issues in case another input format is supported.

Reply:
Done